pyro/generic/generic.py

```diff
@@ -55,36 +61,42 @@ def pyro_backend(*aliases, **new_backends):
 _ALIASES = {
     'pyro': {
         'pyro': 'pyro',
-        'distributions': 'pyro.distributions',
+        'distributions': 'pyro.generic.distributions',
```

Comment:
Can you explain the advantage of adding a new `pyro.generic.distributions` module? I don't see why this is necessary. I'd even prefer to add a pyro/distributions/constraints.py and pyro/distributions/transforms.py that merely import torch stuff, rather than create an extra `pyro/generic/distributions`

Reply:
> Can you explain the advantage of adding a new pyro.generic.distributions module? I don't see why this is necessary. I'd even prefer to add a pyro/distributions/constraints.py and pyro/distributions/transforms.py that merely import torch stuff

There's no advantage at all. I think we can instead create `pyro.distributions.contstraints` and `pyro.distributions.transforms` or even import them as separate modules.